src/main/java/org/jabref/gui/commonfxcontrols/FieldFormatterCleanupsPanelViewModel.java

```diff
@@ -28,6 +30,7 @@
     private final ObjectProperty<Formatter> selectedFormatterProperty = new SimpleObjectProperty<>();
 
     public FieldFormatterCleanupsPanelViewModel() {
+        availableFieldsProperty.sort(Comparator.comparing(Field::getDisplayName));
```

Comment:
Nitpick: In this case, it doesn't change anything, but you can use a [SortedList](https://openjfx.io/javadoc/16/javafx.base/javafx/collections/transformation/SortedList.html#%3Cinit%3E(javafx.collections.ObservableList,java.util.Comparator)) to wrap the `ObservableList` in instead. If at some time in the future the underlying data structure (`FieldFactory.getCommonFields()`) becomes observable this menu will be kept sorted.

Reply:
Emmm... you mean make `availableFieldsProperty` as `SortedList` ?

Reply:
Oups, I might have commented on the wrong line

Reply:
It seems that the Comparator of the sortedList must be chosen at definition? `UnsupportedOperationException` will be throw n if I sort without Comparator. So i wonder whether there is no need to sort in `FieldFormatterCleanupsPanelViewModel`.

Reply:
I am not sure that I understand you. I believe [setComparator](https://openjfx.io/javadoc/16/javafx.base/javafx/collections/transformation/SortedList.html#setComparator(java.util.Comparator)) should work, but why do you want to set it in the constructor?

Reply:
I mean should we still use `availableFieldsProperty.sort(Comparator.comparing(Field::getDisplayName));` after using a sortList ?

Reply:
I can't find a good reference for it, but no, that should not be needed. `SortedList` should keep the list sorted based on the `Comparator` as long as you are making changes that fire change events (e.g., add, remove).

Reply:
OK. Very thanks for your help!

Reply:
[This](https://stackoverflow.com/questions/27806814/javafx-sortedlist-listen-the-list-changes-and-the-list-items-updated-event#tab-top) is an example of when it must be dealt with differently.

And thank you for including the nitpick 👍